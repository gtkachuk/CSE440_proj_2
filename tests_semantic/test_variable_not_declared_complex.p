program testVariableNotDeclaredComplex;

class testVariableNotDeclaredComplex
BEGIN

   VAR 
      value : integer;
      
FUNCTION testVariableNotDeclaredComplex;
BEGIN
   value := 1 + 2 + undeclaredVariable
END

END
.

