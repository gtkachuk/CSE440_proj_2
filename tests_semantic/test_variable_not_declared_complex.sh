#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_variable_not_declared_complex.p
