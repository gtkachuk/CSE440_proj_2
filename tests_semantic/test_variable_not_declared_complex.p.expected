ERROR:12:Variable "undeclaredVariable" not declared
Errors detected. Exiting.
